src/lib.rs

```diff
@@ -37,22 +55,30 @@ mod manifest_store;
 mod mem_table;
 mod mem_table_flush;
 mod merge_iterator;
-pub mod merge_operator;
+
+mod merge_operator;
+pub use merge_operator::{MergeOperator, MergeOperatorError};
```

Comment:
Nit: Mind clumping the `pub use` calls all together in one section so we have a clear one-stop view of the public APIs? This is just my personal preference, so feel free to disregard.

Reply:
yeah how about all the `pub use` at the top of the file, then `pub mod`s, then the internal modules?

Reply:
Sounds good!